xbmc/interfaces/legacy/ModuleXbmcvfs.h

```diff
@@ -167,6 +167,35 @@ namespace XBMCAddon
     String makeLegalFilename(const String& filename);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+    ///
+    /// \ingroup python_xbmcvfs
+    /// @brief \python_func{ xbmcvfs.translatePath(path)  }
+    /// Returns the translated path.
+    ///
+    /// @param path                  string - Path to format
+    /// @return            Translated path
```

Comment:
Maybe cleanup the spaces there, think can also be Ok to use only 1 space after value name.

Was added by me where I changed to doxygen style, but in retrospect, it is probably better to only use one space :thinking:.